kie-server-parent/kie-server-services/kie-server-services-case-mgmt/src/main/java/org/kie/server/services/casemgmt/CaseManagementServiceBase.java

```diff
@@ -334,7 +334,7 @@ public void removeCommentFromCase(String containerId, String caseId, String comm
     }
 
     public CaseCommentList getComments(String containerId, String caseId, Integer page, Integer pageSize) {
-        Collection<CommentInstance> caseComments = caseService.getCaseComments(caseId, ConvertUtils.buildQueryContext(page, pageSize));
+        Collection<CommentInstance> caseComments = caseService.getCaseComments(caseId, page, pageSize, ConvertUtils.buildQueryContext(page, pageSize));
```

Comment:
These page and pageSize is not needed here as same info is already set on he query context